gh-actions-scripts/check-image-digests.sh

```diff
@@ -19,10 +22,14 @@ for tag in "${tags[@]}"; do
       echo "Checking image digest..."
       if [[ "$docker_digest" != "$release_digest" ]]; then
         echo "CAUTION: Docker image digest does not match released digest!"
+        echo "Digest from dockerhub: $docker_digest"
+        echo "Digest from release:   $release_digest"
         exit 1
       else
         echo "Image digest for $artifact:$tag matches released digest. Continuing..."
       fi
     done < $DIGEST_FILE
+    rm "$DIGEST_FILE"
   fi
+  echo ""
```

Comment:
Maybe add a separator between tags, e.g.
```suggestion
  echo "============="
```